### PR TITLE
docs/installation: added Alpine Linux to distribution list

### PR DESCRIPTION
**Pull request by jminer74**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -58,6 +59,7 @@ Raspbian     `Raspbian testing`_                           ``apt install borgbac
 Ubuntu       `Ubuntu packages`_, `Ubuntu PPA`_             ``apt install borgbackup``
 ============ ============================================= =======
 
+.. _Alpine repository: https://pkgs.alpinelinux.org/packages?name=borgbackup&branch=edge
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: what's the meaning of branch edge?

Comment (jminer74) [reply docs/installation.rst]: We don't need it, can just reference:

https://pkgs.alpinelinux.org/packages?name=borgbackup

**Comment by ThomasWaldmann** [comment]
thanks!

**Comment by ThomasWaldmann** [comment]
do you want to backport this to 1.1-maint branch also so it gets into next 1.1.x release?

**Comment by jminer74** [comment]
Sure. Requested

https://github.com/borgbackup/borg/pull/4415

**Pull request merged**